### PR TITLE
test: pytest adjustments to deal with timeouts

### PR DESCRIPTION
**Pull request by thiagowfx**
Docs: https://pypi.org/project/pytest-timeout/

### CODE REVIEW

**Comment by thiagowfx** [comment]
Motivation: https://github.com/GoogleChromeLabs/chromium-bidi/pull/885

E2E takes the full 20 minutes set by the github action

File: pytest.ini [hunk 1]
```diff
@@ -3,3 +3,6 @@
 addopts = --doctest-modules
 minversion = 7.0
 testpaths = tests
+
+# pytest-timeout
+timeout = 60
```
Comment (OrKoN) [inline pytest.ini:8]: could you please add the units as a comment

Comment (OrKoN) [reply pytest.ini:8]: if it is seconds, let's use a value like 5?

Comment (thiagowfx) [reply pytest.ini:8]: Yes, a smaller value makes sense. I was thinking of 10 or so. It can also be 5. The only reason I started with 60 is to be conservative and not disrupt the CI too much in case it proved to be too low.

I'll decrease it to 10 then.

Comment (OrKoN) [reply pytest.ini:8]: I would start with 5 and increase if needed

**Review by OrKoN — APPROVED** [review]

**Comment by thiagowfx** [comment]
5.0s keeps failing `test_input_performActionsEmitsPointerEvents`.

We could increase the timeout only for that test, but let me instead increase the global timeout to 10 for now.

**Comment by thiagowfx** [comment]
Note that previously there was no timeout at all (at the pytest level).

File: src/bidiMapper/domains/events/EventManager.ts [hunk 2]
```diff
@@ -210,8 +209,8 @@ export class EventManager implements IEventManager {
     eventName: Session.SubscriptionRequestEvent,
     contextId: CommonDataTypes.BrowsingContext | null
   ) {
-    // Enable network domain if user subscribed to any of network events.
-    if (eventName.startsWith(EventManager.#NETWORK_DOMAIN_PREFIX)) {
+    // Enable network domain if user subscribed to any network event.
```
Comment (OrKoN) [inline src/bidiMapper/domains/events/EventManager.ts:212]: why these changes are in this PR? (looks like added by mistake)

**Comment by OrKoN** [comment]
`test_input_performActionsEmitsPointerEvents`: judging by the name it probably should not take too long. Perhaps it relies on high fixed timeouts?

**Comment by OrKoN** [comment]
`test_input_performActionsEmitsPointerEvents`: judging by the name it probably should not take too long. Perhaps it relies on high fixed timeouts?

File: src/bidiMapper/domains/events/EventManager.ts (hunk 2 above)
Comment (thiagowfx) [reply src/bidiMapper/domains/events/EventManager.ts:212]: ooops, committed to the wrong branch
